pkgs/kubernetes-sigs/controller-tools/controller-gen/registry.yaml

```diff
@@ -0,0 +1,11 @@
+packages:
+  - type: go_install
+    name: kubernetes-sigs/controller-tools/controller-gen
+    aliases:
+      - name: kubernetes-sigs/kubebuilder/controller-gen
```

Comment:
Why is this alias needed?
`aliases` is used when the repository full name is renamed or we change the package name keeping the compatibility.

Reply:
It's because `controller-gen` is considered as a subproject of `kubebuilder` and normally it's accompanied with it:
https://book.kubebuilder.io/reference/controller-gen.html
so I thought it's convenient for some users who think it's under `kubebuilder`. But if you think otherwise, feel free to remove it.

Reply:
I think the same package should have only one name, and `aliases` should be used only to keep the compatibility.
Users may be confused what's the difference of `kubernetes-sigs/controller-tools/controller-gen` and `kubernetes-sigs/kubebuilder/controller-gen` and which package they should use.

On the other hand, it is useful to allow to search packages with additional words.
To do this, I'll add a new field `search_words`. https://github.com/aquaproj/aqua/pull/972

Reply:
Thanks. I agree. Updated with the new `search_words` field.

Reply:
Oh, sorry. I forgot to update JSON Schema.
https://github.com/aquaproj/aqua/tree/main/json-schema
https://github.com/aquaproj/aqua-registry/runs/7149219266?check_suite_focus=true

Reply:
Updated. https://github.com/aquaproj/aqua/pull/974